mlflow/server/handlers.py

```diff
@@ -67,6 +67,9 @@
     DeleteRegisteredModelTag,
```

Comment:
Do I need to manually populate the `RegisteredModel` and `ModelVersion` payloads here or is it automatically done? Don't see a similar example with tags so assuming it's done elsewhere.

Reply:
Moved aliases to use foreignKey on `RegisteredModel` and manually populate `ModelVersion` payloads.